packages/storefront-lwc/src/modules/commerce/cart/cart.html

```diff
@@ -0,0 +1,204 @@
+<!--
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+<template>
+    <div role="main" id="maincontent">
+
+
+        <div class="cart-error-messaging cart-error">
+        </div>
+
+        <div class="container">
+            <h1 class="page-title">Your Cart</h1>
+            <div class="row cart-header">
+                <div class="col-sm-4 hidden-xs-down">
+                    <a class="continue-shopping-link" href="/s/RefArch/home?lang=en_US" title="Continue Shopping">
+                        Continue Shopping
+                    </a>
+                </div>
+                <div class="col-sm-3 text-center">
+                    <h2 class="number-of-items">
+                        {products.length} Items
+                    </h2>
+                </div>
+                <div class="col-sm-5 text-right hidden-xs-down">
+                    <div>
+                        <span>Need Help? Call</span>
+                        <span><a class="help-phone-number" href="tel:1-800-555-0199">1-800-555-0199</a></span>
+                    </div>
+                </div>
+            </div>
+            <hr class="no-margin-top">
+        </div>
+
+        <div class="container cart cart-page">
+            <div class="row">
+                <div class="col-md-8">
+                    <template for:each={products} for:item="product" for:index="index">
+                        <div key={product.productId} class="card product-info  uuid-1ef92777cbf6c7daaaafdd1905">
+                            <div class="line-item-header">
+                                <div class="line-item-name">
+                                    {product.productName}
+                                </div>
+                                <div class="remove-line-item d-lg-none">
+                                    <button type="button" class="remove-btn remove-product btn btn-light" data-toggle="modal" data-target=".cart.cart-page #removeProductModal" data-pid="701642852674M" data-name="Wide Leg Pant" data-action="/on/demandware.store/Sites-RefArch-Site/en_US/Cart-RemoveProductLineItem" data-uuid="8e56221845f1932db670ca0fb8" aria-label="Remove product Wide Leg Pant">
```

Comment:
Do we need all of these data attributes still? Looks like this is just copy and pated form SFRA. 

Aria-label shouldn't need to be hardcoded.

Reply:
Delete from Cart is not implemented yet. We will clean it up when we work on that.

Reply:
Cleaned